testcases/OpTestSTB.py

```diff
@@ -0,0 +1,623 @@
+                print BMC_CONST.SECURE_MODE_ON
+                return BMC_CONST.SECURE_MODE_ON
+            elif l_secure_mode_status == BMC_CONST.OFF:
+                print BMC_CONST.SECURE_MODE_OFF
+                return BMC_CONST.SECURE_MODE_OFF
+        else:
+            l_msg = "Secure mode enable/disable status not seen"
+            raise OpTestError(l_msg)
+
+    ##
+    # @brief This function collects Trusted boot mode information from OPAL
+    # messagelog and if found determines whether it is 'on' or 'off'.
+    # Returns TRUSTED_MODE_ON for Trusted boot mode 'on'
+    # Returns TRUSTED_MODE_OFF for Trusted boot mode 'off'
+    # Raises OpTestError when no information found
+    #
+    def trusted_boot_mode(self):
+        l_trusted_mode = self.string_get(self.l_data,
+                                         BMC_CONST.TRUSTED_MODE)
+        if l_trusted_mode:
+            l_trusted_mode_status = self.status_finder(l_trusted_mode,
+                                                       BMC_CONST.ON,
+                                                       BMC_CONST.OFF)
+            if l_trusted_mode_status == BMC_CONST.ON:
+                print BMC_CONST.TRUSTED_MODE_ON
+                return BMC_CONST.TRUSTED_MODE_ON
+            elif l_trusted_mode_status == BMC_CONST.OFF:
+                print BMC_CONST.TRUSTED_MODE_OFF
+                return BMC_CONST.TRUSTED_MODE_OFF
+        else:
+            l_msg = "Trusted mode enable/disable status not seen"
+            raise OpTestError(l_msg)
+
+    ##
+    # @brief This function collects CAPP measured information from OPAL
+    # messagelog and if found determines whether it is 'measured' or
+    # 'not measured'.
+    # Returns CAPP_PAR_MEASURED for CAPP partition measured
+    # Returns CAPP_PAR_NOT_MEASURED for CAPP partition not measured
+    # Raises OpTestError when no information found
+    #
+    def capp_measured(self):
+        l_capp_measured = self.string_get(self.l_data,
+                                          BMC_CONST.CAPP_MEASURED)
+        l_capp_not_measured = self.string_get(self.l_data,
+                                              BMC_CONST.TB_SKIPPED)
+        if l_capp_measured:
+            print BMC_CONST.CAPP_PAR_MEASURED
+            return BMC_CONST.CAPP_PAR_MEASURED
+        elif l_capp_not_measured:
+            print BMC_CONST.CAPP_PAR_NOT_MEASURED
+            return BMC_CONST.CAPP_PAR_NOT_MEASURED
+
+        l_msg = "CAPP partition measured details not seen"
+        raise OpTestError(l_msg)
+
+    ##
+    # @brief This function collects BOOTKERNEL measured information from OPAL
+    # messagelog and if found determines whether it is 'measured' or
+    # 'not measured'.
+    # Returns BOOTKERNEL_PAR_MEASURED for BOOTKERNEL partition measured
+    # Returns BOOTKERNEL_PAR_NOT_MEASURED for BOOTKERNEL partition not measured
+    # Raises OpTestError when no information found
+    #
+    def bootkernel_measured(self):
```

Comment:
Change function name to opal_get_bootkernel_measure_info. and just return True/ False for measure/not-measured cases.